modules/apigee_edge_teams/apigee_edge_teams.install

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @file
+ * Copyright 2019 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 2 as published by the
+ * Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+use Apigee\Edge\Utility\OrganizationFeatures;
+
+/**
+ * @file
+ * Install, update and uninstall functions for Apigee Edge Teams.
+ */
+
+/**
+ * Implements hook_requirements().
+ */
+function apigee_edge_teams_requirements($phase) {
+  $requirements = [];
+
+  if ($phase == 'install' || $phase == 'runtime') {
```

Comment:
If the phase is runtime, here is what I see in status report:

<img width="884" alt="team-error" src="https://user-images.githubusercontent.com/613093/69388660-88dfc380-0c7e-11ea-83b4-19545f61607e.png">

If it is runtime, we should add additional text to tell user what needs to be done:

> The Apigee Edge Teams module functionality will not work when connecting to an Edge hybrid org and should be uninstalled.

Reply:
I've updated the messages so that on install:
_The Apigee Edge Teams module functionality is not available for your org because [Edge company APIs are not supported in Apigee hybrid orgs](https://docs.apigee.com/hybrid/compare-hybrid-edge#unsupported-apis)._

And on runtime:
_The Apigee Edge Teams module functionality is not available for your org and should be uninstalled, because [Edge company APIs are not supported in Apigee hybrid orgs](https://docs.apigee.com/hybrid/compare-hybrid-edge#unsupported-apis)._